cli/hzn.go

```diff
@@ -186,14 +186,14 @@ Environment Variables:
 	exAgbotDPPatOrg := exAgbotDelPatCmd.Arg("patternorg", msgPrinter.Sprintf("The organization of the pattern to remove.")).Required().String()
 	exAgbotDPPat := exAgbotDelPatCmd.Arg("pattern", msgPrinter.Sprintf("The name of the pattern to remove.")).Required().String()
 	exAgbotDPNodeOrg := exAgbotDelPatCmd.Arg("nodeorg", msgPrinter.Sprintf("The organization of the nodes that should be searched. Defaults to patternorg.")).String()
-	exAgbotListPolicyCmd := exAgbotCmd.Command("listbusinesspol", msgPrinter.Sprintf("Display the business policies that this agbot is serving."))
-	exAgbotPol := exAgbotListPolicyCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to list serving business policies for.")).Required().String()
-	exAgbotAddPolCmd := exAgbotCmd.Command("addbusinesspol", msgPrinter.Sprintf("Add this business policy to the list of policies this agbot is serving. Currently only support adding all the business polycies from an organization."))
-	exAgbotAPolAg := exAgbotAddPolCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to add the business policy to.")).Required().String()
-	exAgbotAPPolOrg := exAgbotAddPolCmd.Arg("policyorg", msgPrinter.Sprintf("The organization of the business policy to add.")).Required().String()
-	exAgbotDelPolCmd := exAgbotCmd.Command("removebusinesspol", msgPrinter.Sprintf("Remove this business policy from the list of policies this agbot is serving. Currently only support removing all the business polycies from an organization."))
-	exAgbotDPolAg := exAgbotDelPolCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to remove the business policy from.")).Required().String()
-	exAgbotDPPolOrg := exAgbotDelPolCmd.Arg("policyorg", msgPrinter.Sprintf("The organization of the business policy to remove.")).Required().String()
+	exAgbotListPolicyCmd := exAgbotCmd.Command("listdeploymentpol", msgPrinter.Sprintf("Display the deployment policies that this agbot is serving."))
+	exAgbotPol := exAgbotListPolicyCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to list serving deployment policies for.")).Required().String()
+	exAgbotAddPolCmd := exAgbotCmd.Command("adddeploymentpol", msgPrinter.Sprintf("Add this deployment policy to the list of policies this agbot is serving. Currently only support adding all the deployment policies from an organization."))
+	exAgbotAPolAg := exAgbotAddPolCmd.Arg("agbot", msgPrinter.Sprintf("The agbot to add the deployment policy to.")).Required().String()
+	exAgbotAPPolOrg := exAgbotAddPolCmd.Arg("policyorg", msgPrinter.Sprintf("The organization of the deployment policy to add.")).Required().String()
+	exAgbotDelPolCmd := exAgbotCmd.Command("removedeploymentpol", msgPrinter.Sprintf("Remove this deployment policy from the list of policies this agbot is serving. Currently only support removing all the deployment policies from an organization."))
```

Comment:
Could you also add alias for listdeploymentpol, adddeploymentpol and removedeploymentpol to support backward compatibility?

Reply:
Added